app/templates/rounds.hbs

```diff
@@ -6,4 +6,4 @@
     Friends Rounds
   {{/tabs.tab}}
 {{/paper-tabs}}
-{{liquid-outlet}}
\ No newline at end of file
+{{liquid-outlet class="md-padding"}}
```

Comment:
Is this given to us by `ember-paper`? Can you specify what directing the padding/margin does in? Example: `md-padding-top`